Makefile

```diff
@@ -35,20 +35,20 @@ SOURCE_DIR                 := $(CURDIR)/build/package/rpm
 SOURCE_TARBALL_FILENAME    := go-fdo-server-$(VERSION).tar.gz
 SOURCE_TARBALL             := $(SOURCE_DIR)/${SOURCE_TARBALL_FILENAME}
 $(SOURCE_TARBALL):
+	@echo "Creating source tarball..."
 	git archive --prefix=go-fdo-server-$(VERSION)/ --format=tar.gz HEAD > $(SOURCE_TARBALL)
 
 .PHONY: source-tarball
 source-tarball: $(SOURCE_TARBALL)
 
-GO_VENDOR_TOOLS_FILE_NAME  := go-vendor-tools.toml
-GO_VENDOR_TOOLS_FILE       := $(SOURCE_DIR)/$(GO_VENDOR_TOOLS_FILE_NAME)
 VENDOR_TARBALL_FILENAME    := go-fdo-server-$(VERSION)-vendor.tar.bz2
 VENDOR_TARBALL             := $(SOURCE_DIR)/$(VENDOR_TARBALL_FILENAME)
 $(VENDOR_TARBALL):
+	@echo "Creating vendor tarball..."
 	rm -rf vendor; \
-	command -v go_vendor_archive || sudo dnf install -y go-vendor-tools python3-tomlkit; \
-	go_vendor_archive create --config $(GO_VENDOR_TOOLS_FILE) --write-config --output $(VENDOR_TARBALL) .; \
-	rm -rf vendor;
+	go mod vendor; \
+	tar -cjf $(VENDOR_TARBALL) vendor/; \
+	rm -rf vendor
```

Comment:
The problem is the spec file depending on `go-vendor-tools`, not its use in the Makefile. We can keep the current vendor tarball generation I believe